src/ingest_validation_tools/table-schemas/donor.yaml

```diff
@@ -2,31 +2,34 @@
 # The full description of Donor metadata is at
 # https://portal.hubmapconsortium.org/docs/donor
 
+# Most definitions taken from:
+# https://ncit.nci.nih.gov/ncitbrowser/pages/home.jsf?version=20.11e
+
 fields:
 # -
 #   name: age_unit
 #   description: TODO
-# -
-#   name: age_value
-#   description: TODO
-# -
-#   name: blood_type
-#   description: TODO
+-
+  name: age_value
+  description:  The time elapsed since birth. 
+-
+  name: blood_type
+  description:  'The specific reaction pattern of erythrocytes of an individual to the antisera of one blood group; e.g., the ABO blood group consists of four major blood types: O, A, B, and AB. This classification depends on the presence or absence of two major antigens: A or B. Type O occurs when neither is present and type AB when both are present. The blood type is the genetic phenotype of the individual for one blood group system and may be determined using different antisera available for testing.' 
 # -
 #   name: body_mass_index_unit
 #   description: TODO
-# -
-#   name: body_mass_index_value
-#   description: TODO
-# -
-#   name: cause_of_death
-#   description: TODO
+-
+  name: body_mass_index_value
+  description: An individual's weight in kilograms divided by the square of the height in meters. 
+-
+  name: cause_of_death
+  description:  The circumstance or condition that results in the death of a living being.
 # -
 #   name: death_event
 #   description: TODO
-# -
-#   name: height_unit
-#   description: TODO
+-
+  name: height_unit
+  description:  The vertical measurement or distance from the base to the top of a subject or participant. 
 # -
 #   name: height_value
 #   description: TODO

```

Comment:
```suggestion
#   description:  The  body length of a subject from feet to the head.

```

Reply:
No: The `unit` is different than the `value`.